website/notifications/tasks.py

```diff
@@ -34,6 +34,8 @@ def send_users_email(send_type):
                 mails.send_mail(
                     to_addr=user.username,
                     mimetype='html',
+                    node=AbstractNode.load(sorted_messages['children'].keys()[0]),
```

Comment:
One concern of build node in this way is that it only gives the first node in the message. But actually for comments email which is in the form of digest email or notification email, it sends out all the actions within a time period which can contains action not only to a single node but multiple nodes. Only offer user access to one of the node won't change the result that he will still receive notification from other node even if he change the setting for the node.

Reply:
Excellent catch @chennan47. Here's an example of the scenario described: 

![screen shot 2018-03-13 at 1 45 01 pm](https://user-images.githubusercontent.com/7913604/37359772-da4d66ce-26c4-11e8-9545-84cab57f2099.png)

This PR would change the above screenshot to have the following footer text: "To change how often you receive emails, visit this project's settings (link to "dash" project's settings) for emails about this project, or your user settings (link to user settings) to manage default email settings."

As Nan points out, only linking to one of the three projects mentioned in the email is undesirable and not a great improvement.

We could add the "To change how often you receive emails, visit this project's settings (link to the appropriate project's settings) for emails about this project" text to each section of the email, but I think that might be overkill (and might be a larger change? @Johnetordoff can check me on that). 

Alternatively, we could defer adding the project settings link to notification emails that reference multiple projects. 

Pinging @saradbowman and @sloria for input.